docs/3.0-api-changes.xml

```diff
@@ -5,6 +5,16 @@
   This is a list of API changes in libblockdev 3.0 with suggested replacements.
   </para>
 
+  <simplesect><title>Library</title>
+  <para>
+    <literal>bd_switch_init_checks()</literal> has been removed. Checks for runtime dependencies during libblockdev initialization are now disabled and plugins will be loaded even if a runtime dependency is missing.
```

Comment:
I'd make this **bold** because this is a quite a big change in the default behavior.

Reply:
Ok. Big changes like these will be documented in release notes too.